lib/ansible/modules/storage/infinidat/infini_export.py

```diff
@@ -0,0 +1,190 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# (c) 2016, Gregory Shulov (gregory.shulov@gmail.com)
+#
+# This file is part of Ansible
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible. If not, see <http://www.gnu.org/licenses/>.
+
+DOCUMENTATION = '''
+---
+module: infini_export
+version_added: 2.3
+short_description: Create, Delete or Modify NFS Exports on Infinibox
+description:
+    - This module creates, deletes or modifies NFS exports on Infinibox.
+options:
+  name:
+    description:
+      - Export name. Should always start with C(/). (ex. name=/data)
+    aliases: ['export', 'path']
+    required: true
+  state:
+    description:
+      - Creates/Modifies export when present and removes when absent.
+    required: false
+    default: "present"
+    choices: [ "present", "absent" ]
+  inner_path:
+    description:
+      - Internal path of the export
+    default: "/"
+  client_list:
+    description:
+      - List of dictionaries with client entries. See examples.
+        Check infini_export_client module to modify individual NFS client entries for export
+    default: "All Hosts(*), RW, no_root_squash: True"
+    required: false
+  filesystem:
+    description:
+      - Name of exported file system
+    required: true
+extends_documentation_fragment:
+    - infinibox
+'''
+
+EXAMPLES = '''
+- name: Export bar filesystem under foo pool as /data
+  infini_export: name=/data01 filesystem=foo user=admin password=secret system=ibox001
```

Comment:
We are standerdising on multiline YAMl examples, please rewrite in the same form as your `Export and specify client list explicitly` example.

This comment may apply to other examples in your new modules as well.